content/en/blog/2024/ambient-reaches-beta/index.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Istio’s new sidecar-less ‘Ambient Mode’ reaches Beta in version 1.22"
+description: The latest release from Istio brings service mesh Layer 4 & 7 features to production readiness without sidecars.
+publishdate: 2024-05-14
+attribution: "Lin Sun (Solo.io), for the Istio Steering and Technical Oversight Committees"
+keywords: [Istio Day,Istio,conference,KubeCon,CloudNativeCon]
+---
+
+Istio ambient service mesh was [announced in September 2022](/blog/2022/introducing-ambient-mesh/) as an experimental branch
+that introduced a new data plane mode in Istio that did not require sidecars, offering a simplified operational experience of
+Istio. After 20 months of hard work and collaboration within the Istio community, with contributions from Solo.io, Google,
+Microsoft, Intel, Aviatrix, Huawei, IBM, Red Hat, and others, we are excited to announce that ambient mode has reached Beta in
+version 1.22! The beta release of 1.22 indicates the features of ambient mode are now stable and ready for production workloads
+with appropriate cautions.  This is a huge milestone for Istio, bringing both Layer 4 and Layer 7 mesh features to production
+readiness without sidecars.
+
+## Why ambient mode?
+
+We listened to the feedback from Istio users and observed a growing demand for mesh capabilities for their applications but who found the resource overhead and operational complexity of sidecars hard to overcome.
+functions for their applications and found the complexity and overhead of sidecars from service meshes hard to overcome.
```

Comment:
I think you need to collapse this setence into above, my comment applied to both. Alse shouldn't be a newline here.